applications/nrf_desktop/README.rst

```diff
@@ -1182,14 +1182,14 @@ Changing interrupt priority
 You can edit the DTS files to change the priority of the peripheral's interrupt.
 This can be useful when :ref:`adding a new custom board <porting_guide_adding_board>` or whenever you need to change the interrupt priority.
 
-The ``interrupts`` property is an array, where meaning of each element is defined by the specification of the interrupt controller.
+The ``interrupts`` property is an array, where the meaning of each element is defined by the specification of the interrupt controller.
 These specification files are located at :file:`zephyr/dts/bindings/interrupt-controller/` DTS binding file directory.
 
 For example, for nRF52840 the file is :file:`arm,v7m-nvic.yaml`.
 This file defines ``interrupts`` property in the ``interrupt-cells`` list.
-In case of nRF52840, it contains two elements: ``irq`` and ``priority``.
+For the nRF52840, it contains two elements: ``irq`` and ``priority``.
```

Comment:
We use "the" with the name of the SoC if it is followed by `DK`, `SoC`, and so on. See https://projecttools.nordicsemi.no/confluence/display/MAR/Product+names . 
```suggestion
For nRF52840, it contains two elements: ``irq`` and ``priority``.
```